controllers/tempest_controller.go

```diff
@@ -208,7 +208,7 @@ func (r *TempestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 		helper,
 		serviceLabels,
 		instance.Spec.StorageClass,
-		instance.Spec.Parallel,
+		0, // default naming
```

Comment:
Can you please simply the code so that there are no two calls of the EnsureLogsPVCExists() function?